images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -134,6 +134,7 @@ $azcopyVersion = Run-Command "azcopy --version" | Take-Part -Part 2
 $zstdVersion = Run-Command "zstd --version" | Take-Part -Part 1 -Delimiter "v" | Take-Part -Part 0 -Delimiter ","
 $bazelVersion = Run-Command "bazel --version" | Take-Part -Part 0 -Delimiter "-"
 $bazeliskVersion = Run-Command "brew list bazelisk --versions"
+$awsIamAuthenticatorVersion = Run-Command "aws-iam-authenticator --version"

```

Comment:
0. Let's move it under this line https://github.com/actions/virtual-environments/blob/55d0c3bed6d1c8e1480a85a2d4965add4940b9fc/images/macos/software-report/SoftwareReport.Generator.ps1#L189
1. should be "aws-iam-authenticator version" 
2. the version output is {"Version":"v0.5.2","Commit":"292b9b82df69b87af962b92485b254d9f4b10f00"} which is not suitable here. Need to implement a function to output only `0.5.2` somewhere in this file https://github.com/actions/virtual-environments/blob/main/images/macos/software-report/SoftwareReport.Common.psm1